contao/templates/dcbe_general_parentView.html5

```diff
@@ -9,13 +9,6 @@
 
 				<?php if(strlen($this->breadcrumb)) echo $this->breadcrumb; ?>
 
-				<?php if ($this->select): ?>
```

Comment:
We have:
* dcbe_general_grouping - used in ParentView and ListView when grouping is active.
* dcbe_general_parentView - used in ParentView when no grouping active.
* dcbe_general_listView_sorting - used in ListView when no grouping but sorting is active.
* dcbe_general_listView - used in ListView when neither grouping nor sorting is active.
* dcbe_general_treeview - used in TreeView.

Therefore I simply fail to see where this change fixes MetaModels/core#885 and more important, keeps functionality in parent view.